pkg/mcs/resourcemanager/server/apis/v1/api.go

```diff
@@ -223,6 +233,7 @@ func (s *Service) deleteResourceGroup(c *gin.Context) {
 	keyspaceID := rmserver.ExtractKeyspaceID(keyspaceIDValue)
 	if err := s.manager.DeleteResourceGroup(keyspaceID, c.Param("name")); err != nil {
 		c.String(http.StatusNotFound, err.Error())
```

Comment:
Should this error also use `errs.ErrResourceGroupNotExists`?

Reply:
seem DeleteResourceGroup will return ErrKeyspaceNotExists

Reply:
> seem DeleteResourceGroup will return ErrKeyspaceNotExists

so replace it with `c.String(http.StatusNotFound, errs.ErrResourceGroupNotExists.FastGenByArgs(groupName).Error())`

Reply:
Shall we change the internal error? DeleteResourceGroup also has an error in the storage layer.